### DISCUSSION TITLE
Existing Rust alternative utils

### DISCUSSION THREAD

**Discussion by baarkerlounger** [uutils/coreutils#4337] [General]
Does this project in any way make use of, collaborate with, have a view on existing rust util implementations? For example [ripgrep](https://github.com/BurntSushi/ripgrep), [batcat](https://github.com/sharkdp/bat), [fd](https://github.com/sharkdp/fd) etc?

**Comment by tertsdiepraam** [comment]
Not directly, but we use some of their libraries, like ripgrep's [`walkdir`](https://github.com/BurntSushi/walkdir)

Other examples:
- `clap` is also used by many of those utilities. The same goes for other foundational libraries (like `unicode-width`)
- `rust-term-grid` is a library from `exa` (we now forked this library)
- `ripgrep` and anything else by BurntSushi is a treasure trove of excellent libraries (`bstr`, `walkdir`, `aho-corastick`, etc.)
- We use `hyperfine`to benchmark the utils, which is written by the author of `bat` and `fd` if that counts 😄
